auto_landing/include/auto_landing/tracking.hpp

```diff
@@ -0,0 +1,35 @@
+#include<ros/ros.h>
+
+#include<geometry_msgs/Twist.h>
+#include<geometry_msgs/PoseStamped.h>
+
+#include<nav_msgs/Odometry.h>
+
+#include<std_srvs/Trigger.h>
+
+namespace ariitk::auto_landing {
+    
+class Tracking {
+    public:
+        Tracking();
+        ~Tracking(){};
+        void init(ros::NodeHandle& nh);
+        void run();
+
+    private:
+    
+        void pose_clbk(const nav_msgs::Odometry& msgs);
+        bool server_clbk(std_srvs::Trigger::Request &req, std_srvs::Trigger::Response &resp);
+    
+        geometry_msgs::PoseStamped setpt_;
+        nav_msgs::Odometry hk_pose_;
```

Comment:
if the variable name is really big then make it small else use the proper name, it is really hard to understand.
like: what is hk_pose_... or ...